eng/pipelines/runtime.yml

```diff
@@ -72,6 +72,7 @@ jobs:
     - Linux_musl_arm64
     - Linux_musl_x64
     - OSX_arm64
+    - Tizen_armel
```

Comment:
This will only build coreclr for Tizen, do we want to build other parts of the product as well?

Reply:
It should be enough to just build CoreCLR, for now at least.

The motivation for adding this coverage is to catch Tizen specific build breaks. The Tizen specific build breaks have been pretty much always in CoreCLR.